_episodes/06-debugging.md

```diff
@@ -0,0 +1,309 @@
+---
+title: "Debugging"
+teaching: 0
+exercises: 0
+questions:
+- "How can I handle errors/warnings?"
+objectives:
+- "Fix a broken recipe"
+keypoints:
+- "The log files contain information about errors and warnings."
```

Comment:
What about: "There are 3 different kinds of log files: main_log, main_log_debug, and log.txt for each diagnostic script."

Reply:
your suggestion is added.